src/Components/Web.JS/src/Platform/Circuits/CircuitStreamingInterop.ts

```diff
@@ -49,3 +50,23 @@ export function sendJSDataStream(connection: HubConnection, data: ArrayBufferVie
         }
     }, 0);
 };
+
+async function getNextChunk(data: ArrayBufferView | Blob, position: number, nextChunkSize: number): Promise<Uint8Array> {
+    if (data instanceof Blob) {
+        return await getChunkFromBlob(data, position, nextChunkSize);
+    } else {
+        return getChunkFromArrayBufferView(data, position, nextChunkSize);
+    }
+}
+
+async function getChunkFromBlob(data: Blob, position: number, nextChunkSize: number): Promise<Uint8Array> {
+    const chunkBlob = data.slice(position, Math.min(position + nextChunkSize, data.size));
```

Comment:
Do we need to be checking `Math.min(position + nextChunkSize, data.size)` here? It looks like the calling code already picks a `nextChunkSize` that won't exceed the total size of the data.

Not a big deal but maybe could slightly simplify it if so :)

Reply:
Ahh great catch, thanks! Updating..